sky/utils/schemas.py

```diff
@@ -334,9 +334,21 @@ def get_service_schema():
                     'qps_lower_threshold': {
                         'type': 'number',
                     },
+                    'target_qps_per_replica': {
+                        'type': 'number',
+                    },
                     'auto_restart': {
                         'type': 'boolean',
                     },
```

Comment:
Add a TODO to remove these deprecated configs after 2 minor releases. Same for qps_lower_threshold